themes/mlibrary_new/custom.php

```diff
@@ -415,27 +412,27 @@ function mlibrary_new_alt_text($item) {
  * Retrieve a thumnail image for a video item type
  *  It is not used in this installation, but it can be used in the future.
  **/
-function mlibrary_new_exhibit_builder_video_attachment($item) {
+function mlibrary_new_exhibit_builder_video_attachment($item,$alt) {
 $remove[] = "'";
 	$elementids_youtube_video = metadata($item, array('Item Type Metadata', 'Video_embeded_code'), array('no_escape'=>true,'all'=>true));
 	$elementvideos_kultura_VCM = metadata($item, array('Item Type Metadata', 'video_embeded_code_VCM'),array('no_escape'=>true, 'all'=>true));
-	$alt_text = mlibrary_new_alt_text($item);
+	//$alt_text = mlibrary_new_alt_text($item);
         if (!empty($elementids_youtube_video)) {
 		foreach ($elementids_youtube_video as $elementid_youtube_video) {
 			$videoid = str_replace($remove, "", $elementid_youtube_video);
 			if (!empty($videoid)) {
-				$video_thumnail_image = "<img src='//i4.ytimg.com/vi/".$videoid."/default.jpg' style='width:200px; height:152px' alt='".$alt_text."'/>";
+				$video_gallery_image = "<img src='//i.ytimg.com/vi/".$videoid."/maxresdefault.jpg' style='width:326px; height:326px' alt=$alt/>";
 			}
 		}
         }//if
         elseif (!empty($elementvideos_kultura_VCM)) {
   	        $data = $elementvideos_kultura_VCM[0];
 		preg_match('/\/entry_id\/([a-zA-Z0-9\_]*)?/i', $data, $match);
                 $partnerId = 1038472;
-                $video_thumnail_image = '<img src="//cdn.kaltura.com/p/'.$partnerId.'/thumbnail/entry_id/'.$match[1].'/width/400/height/400/type/1/quality/100/" alt="'.$alt_text.'"/>';
+                $video_gallery_image = '<img src="//cdn.kaltura.com/p/'.$partnerId.'/thumbnail/entry_id/'.$match[1].'/width/400/height/400/type/1/quality/100/"/>';
         }//if
-        $html = exhibit_builder_link_to_exhibit_item($video_thumnail_image,'',$item);
-  return $html;
+        //$html = exhibit_builder_link_to_exhibit_item($video_gallery_image,'',$item);
```

Comment:
This should probably be deleted rather than left as a comment.

Reply:
Agree, I will check if it is the last updated file.